packages/terra-form-field/docs/MigrationGuide.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from terra-form's Field Component to terra-form-field
```

Comment:
Since you included this migration guide, is it worth adding it to terra-site as well so that it's easy to see? Or at least provide a link to it?

Reply:
Adding on to this are we considering the original Field deprecated? Do we need to post a banner or message on the original Field doc page indicating this or will we do this at a later time? 

I imagine all development on the original Field will stop when this is released to prevent inconsistency and technical debt

Reply:
@bjankord @mjhenkes

Reply:
> I assume deprecation would come with the deprecation of the terra-form in full.

There's going to be an unknown period of time where we have released form packages such as `terra-form-field` but have not released every broken out package from terra-form. During this time there will be two components ( `terra-form-field` / `terra-form/lib/field` ). 

My vote is as soon as we have a component we want to encourage people to use we should to deprecate the old component.

Reply:
>My vote is as soon as we release a component we want to encourage people to use we should to deprecate the old component.

Agreed